sql/core/src/main/scala/org/apache/spark/sql/execution/QueryExecution.scala

```diff
@@ -155,6 +155,7 @@ class QueryExecution(val sparkSession: SparkSession, val logical: LogicalPlan) {
       case (null, _) => "null"
       case (s: String, StringType) => "\"" + s + "\""
       case (decimal, DecimalType()) => decimal.toString
+      case (interval, CalendarIntervalType) => interval.toString
```

Comment:
Do we have a test case to capture this change?

Reply:
ok, I'll try to add tests for that.

Reply:
Added: https://github.com/apache/spark/pull/20433/files#diff-b608c1c9b761ffb3a19bf5c0cd971e43R43